router/handler.go

```diff
@@ -94,13 +145,20 @@ func (m *messageHandler) handleError(requestMessage *fimpgo.Message, err error)
 		return nil
 	}
 
+	errMessage := fmt.Sprintf("failed to process message sent to topic %s service %s and type %s: %s",
```

Comment:
Having a structure here instead of string would be much convenient in my opinion.

Reply:
Yes, that is true, however, we need to discuss it with Christer and arrive at a consensus. Consider it a temporary solution that can be easily extended at any time without any harm.